src/ldc/simd.di

```diff
@@ -204,14 +209,44 @@ if(is(typeof(llvmVecType!V)))
     alias BaseType!V T;
     enum llvmT = llvmType!T;
     enum llvmV = llvmVecType!V;
-    enum ir = `
-        %p = bitcast `~llvmT~`* %0 to `~llvmV~`*
-        %r = load `~llvmV~`* %p, align 1
-        ret `~llvmV~` %r`;
+    version (LDC_LLVM_PRE307)
+        enum ir = `
+            %p = bitcast `~llvmT~`* %0 to `~llvmV~`*
+            %r = load `~llvmV~`* %p, align 1
+            ret `~llvmV~` %r`;
+    else
+        enum ir = `
+            %p = bitcast `~llvmT~`* %0 to `~llvmV~`*
+            %r = load `~llvmV~`, `~llvmV~`* %p, align 1
+            ret `~llvmV~` %r`;
 
     alias inlineIR!(ir, V, T*) loadUnaligned;
 }
 
+unittest
```

Comment:
No idea how your testsuite works, adding a `test/ir/Makefile` prolly wouldn't work. Could you move this to an appropriate place?